Twig/CookieGuardExtension.php

```diff
@@ -1,83 +1,62 @@
 <?php
+declare(strict_types=1);
 
 namespace FH\Bundle\CookieGuardBundle\Twig;
 
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\RequestStack;
 
-class CookieGuardExtension extends \Twig_Extension
+final class CookieGuardExtension extends \Twig_Extension
```

Comment:
Dit zijn wel BI puntjes. Zou in theorie kunnen dat iemand dit extend heeft. Zelfde geldt voor method signatures.

Dus mogelijk moeten we nu wel richting een 2.0 toe, ofwel terugdraaien.

Reply:
Kijkend naar https://github.com/search?q=CookieGuardExtension&type=Code en de leeftijd van de Bundle zou het wellicht wel moeten kunnen

Reply:
Versies kosten niets, laten we het goed doen :). Dus in dit geval terugdraaien de beste oplossing, want voegt niet superveel toe natuurlijk.

Reply:
Het is ook geen best practice om Twig Extensions te extenden lijkt me en gezien het huidige gebruik zou dat geen problemen moeten geven. En voegt het wel wat toe...?

Reply:
Het gaat erom dat we semantic versioning goed toepassen. Als we een major release omhoog gaan vanwege deze wijziging, vind ik de meerwaarde beperkt. Dus vandaar mijn voorstel  om de aanpassingen terug te draaien.
Als we de aanpassingen toch door willen voeren moeten we ze opnemen in de volgende major release